extensions/wikia/ImageReview/ImageReviewDatabaseHelper.class.php

```diff
@@ -62,22 +70,22 @@ public function countImagesByState() {
 		$oDB = $this->getDatawareDB( DB_SLAVE );
 
 		$aWhere = [];
-		$aStatesToFetch = array(
+		$aStatesToFetch = [
 			ImageReviewStatuses::STATE_QUESTIONABLE,
 			ImageReviewStatuses::STATE_REJECTED,
 			ImageReviewStatuses::STATE_UNREVIEWED,
 			ImageReviewStatuses::STATE_INVALID_IMAGE,
-		);
+		];
 		$aWhere[] = 'state in (' . $oDB->makeList( $aStatesToFetch ) . ')';
 		$aWhere[] = 'top_200=0';
 
 		// select by reviewer, state and total count with rollup and then pick the data we want out
 		$oResults = $oDB->select(
-			array( 'image_review' ),
-			array( 'state', 'count(*) as total' ),
+			[ 'image_review' ],
+			[ 'state', 'count(*) as total' ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "state" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)